src/sentry/snuba/discover.py

```diff
@@ -735,6 +735,110 @@ def get_facets(query, params, limit=10, referrer=None):
     return results
 
 
+def get_performance_facets(
+    query,
+    params,
+    orderby=None,
+    aggregate_column="duration",
+    aggregate_function="avg",
+    limit=20,
+    referrer=None,
+):
+    """
+    High-level API for getting 'facet map' results for performance data
+
+    Performance facets are high frequency tags and the aggregate duration of
+    their most frequent values
+
+    query (str) Filter query string to create conditions from.
+    params (Dict[str, str]) Filtering parameters with start, end, project_id, environment
+    limit (int) The number of records to fetch.
+    referrer (str|None) A referrer string to help locate the origin of this query.
+
+    Returns Sequence[FacetResult]
+    """
+    with sentry_sdk.start_span(
+        op="discover.discover", description="facets.filter_transform"
+    ) as span:
+        span.set_data("query", query)
+        snuba_filter = get_filter(query, params)
+
+        # Resolve the public aliases into the discover dataset names.
+        snuba_filter, translated_columns = resolve_discover_aliases(snuba_filter)
+
+    # Exclude tracing tags as they are noisy and generally not helpful.
+    # TODO(markus): Tracing tags are no longer written but may still reside in DB.
```

Comment:
Is this still true? Or has retention policy evicted all the tracing tags?

Reply:
Actually not sure about the comment but assumed faceting by trace wouldn't really be helpful since we are looking for performance facets of user conditions, not one-offs